androidApp/src/main/java/com/mbta/tid/mbta_app/android/pages/MapAndSheetPage.kt

```diff
@@ -365,7 +365,7 @@ fun MapAndSheetPage(
 
         if (routeId != null) analytics.tappedVehicle(routeId)
 
-        val newTripFilter = TripDetailsFilter(tripId, vehicle.id, stopSequence)
+        val newTripFilter = TripDetailsFilter(tripId, vehicle.id, stopSequence, true)
```

Comment:
This `selectionLock = true` got lost in #1309; it probably won’t matter, since I doubt we’ll need to release without track this trip, but it’s worth putting back just in case.